src/gui/DatabaseWidget.cpp

```diff
@@ -1215,6 +1215,21 @@ void DatabaseWidget::updateFilePath(const QString& filePath)
         m_fileWatcher.removePath(m_filePath);
     }
 
+#if defined(__linux) || defined(__linux__) || defined(linux)
```

Comment:
Please use the Q_OS_LINUX def that Qt provides or Q_OS_UNIX if this also applies to other *nixes.

Reply:
I will change it to Q_OS_LINUX as statfs is only available on linux.